great_expectations/cli/v012/toolkit.py

```diff
@@ -295,7 +295,7 @@ def create_empty_suite(
     )
     suite = context.create_expectation_suite(expectation_suite_name)
     suite.add_citation(comment="New suite added via CLI", batch_kwargs=batch_kwargs)
-    context.save_expectation_suite(suite, expectation_suite_name)
```

Comment:
What is the difference between `save_expectation_suite` and `update_expectation_suite`?

Reply:
`save_expectation_suite` is an old method that essentially works as `add` (can overwrite existing). `update` takes in the same arguments but can only update an existing object. This is to be consistent across our API:
- `add` - only new objects
- `update` - only existing objects
- `add_or_update` - essentially upsert

Reply:
So why are we using `save_expectation_suite` still?

Reply:
whoops you're right!